cnd/src/swap_protocols/halight/connector.rs

```diff
@@ -94,7 +137,7 @@ impl LndConnectorParams {
 
 fn read_file<T>(path: PathBuf) -> anyhow::Result<T>
 where
-    T: TryFrom<Vec<u8>, Error = Error>,
+    T: TryFrom<Vec<u8>, Error = anyhow::Error>,
```

Comment:
```suggestion
    T: TryFrom<Vec<u8>, Error = Error>,
```
`anyhow::Error` is already imported in this file.

Reply:
I would actually prefer if we never import it because it makes it hard to know, which `Error` it is without jumping to the definition. `anyhow::Error` is still short and avoids any kind of ambiguity.